model/import/Parser/TemplateHeaderParser.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2021 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoQtiItem\model\import\Parser;
+
+use oat\oatbox\service\ConfigurableService;
+use oat\taoQtiItem\model\import\TemplateInterface;
+
+class TemplateHeaderParser extends ConfigurableService
+{
+    public function parse(TemplateInterface $template): array
+    {
+        $csvColumns = $template->getDefinition()['columns'] ?? [];
```

Comment:
Lets move the metadata parsing to this method: https://github.com/oat-sa/extension-tao-itemqti/pull/1751/files#diff-4197dbe0e958d824be0e244e0b15db46a26868068b05291b7e885e48c137bcb6R58

Reply:
It was done